import_test.go

```diff
@@ -115,6 +115,14 @@ func TestImport(t *testing.T) {
 		if m.Layers[0].Digest != d1 {
 			t.Fatalf("unexpected layer hash %s, expected %s", m.Layers[0].Digest, d1)
 		}
+
+		if len(layerAnnotation) != 0 && m.Layers[0].Annotations["ocispec"] != layerAnnotation {
```

Comment:
`checkManifest` is just for quick validation of the manifest. If you want to compare it with something, construct an "expected" manifest and compare the results.